src/db/kv/index/sorted_sets.rs

```diff
@@ -56,16 +56,29 @@ pub async fn check_member(
 /// # Errors
 ///
 /// Returns an error if the operation fails.
-pub async fn put(prefix: &str, key: &str, items: &[(f64, &str)]) -> Result<(), DynError> {
+pub async fn put(
+    prefix: &str,
+    key: &str,
+    items: &[(f64, &str)],
+    expiration: Option<i64>,
```

Comment:
`expiration` is provided in  `minutes` and not documented. However in `index::json` and in `index::sets` we provide expiration in `seconds` . Maybe best to always provide expiration in seconds to avoid confusion.